components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -233,34 +237,41 @@ tweaking their $case/namelist_scream.xml file.
     ${DIN_LOC_ROOT}/atm/scream/tables/revap_table_vals.dat8,
     ${DIN_LOC_ROOT}/atm/scream/tables/vn_table_vals.dat8,
     ${DIN_LOC_ROOT}/atm/scream/tables/vm_table_vals.dat8,
-    ${DIN_LOC_ROOT}/atm/scream/init/spa_file_unified_and_complete_ne4_scream.nc
+    ${DIN_LOC_ROOT}/atm/scream/init/spa_file_unified_and_complete_ne4_20220428.nc,
+    ${DIN_LOC_ROOT}/atm/scream/init/spa_file_unified_and_complete_ne30_20220428.nc
   </input_files>
 
   <!-- Homme control namelist -->
   <ctl_nl>
-    <cubed_sphere_map>0</cubed_sphere_map>
+    <cubed_sphere_map>2</cubed_sphere_map>
     <disable_diagnostics>False</disable_diagnostics>
-    <dt_remap_factor>1</dt_remap_factor>
-    <dt_tracer_factor>1</dt_tracer_factor>
+    <dt_remap_factor>2</dt_remap_factor>
+    <dt_tracer_factor hgrid="ne4np4">6</dt_tracer_factor>
+    <dt_tracer_factor hgrid="ne30np4">6</dt_tracer_factor>
```

Comment:
why not just have one entry for tracer_factor and don't condition on hgrid since you have the same value for all grids?

Reply:
Same with hypervis_subcycle_q?

Reply:
I missed this the  first time I looked.  These are the settings for SL transport.  I thought that wasn't ready yet for EAMxx, so scream v1 should be running with Eulerian transport?

Reply:
Yes, that's correct. We should be using Eulerian transport settings for v1.

Reply:
@mt5555 should I be looking at the defaults for a different section of the Confluence page then?

Reply:
@bartgol , per Peter's question.  I was going to ask, is it possible to have one XML setting be set to another.  In this case, according to the homme recommendations `hypervis_subcycle_q` should be set to match `dt_tracer_factor`.

Reply:
I wasn't asking if we could link different variables to be the same - that seems too complicated and not really necessary. Instead, I was just noticing that you had entries for hypervis_subcycle_q and dt_tracer_factor which were different for different horiz resolution... but you assigned these quantities to the same value regardless of their resolution. Might as well not condition on resolution at all. Would save 2 lines of xml and avoid setting weird precedents for people who don't know what they're doing.

Reply:
Yes, your question just reminded me of something I wanted to ask in general.  We can certainly save two lines by keeping the default as is.  I arranged it this way because in the Confluence page these were one of those namelist options that were supposed to change by resolution.  It just so happen that ne4 and ne30 have the same value, but when we go to ne120+ they should change.  Plus, I was ready to be told that ne4 might be wrong - I took that value from a SCREAMv0-HR ne4 run.  I'm happy either way.  I guess ultimately, I'll wait to hear back from HOMME folks about what changes, if any, are needed for using the Eulerian dycore.  I can consolidate if/when I make those changes.